vtex-apps/store-components
# Feature/product sku names

#### What problem is this solving?
Provide a list of the selected variations for the current product, also provide a source dependency to the `product-summary` 



#### How to test it?
Add the `product-sku-names` block to any block below `store.product` (Product template). For example:

```json
  "store.product": {
    "children": [
      "flex-layout.row#product",
    ]
  },
  "flex-layout.row#product": {
    "children": [
      "product-sku-names"
    ]
  },
```




[Check it working here](https://wender--alssports.myvtex.com/nixon-regulus-watch/p)

#### Screenshots or example usage:
![image](https://user-images.githubusercontent.com/24723/93642867-358ced80-f9d5-11ea-9dad-a5286eb04efd.png)



#### Describe alternatives you've considered, if any.



#### Related to / Depends on



#### How does this PR make you feel? [:link:](http://giphy.com/)



![](put .gif link here - can be found under "advanced" on giphy)
